### ISSUE TITLE
Explain seems to not be working?

### ISSUE THREAD

**Issue by jdatcmd** [commandprompt/pgmanage#129]
```
In the query window, if I type:

SELECT * FROM pg_database;

And then hit the > I get results. However, if I click the explain tab, nothing appears. Further if I type:

EXPLAIN select * from pg_database;

And then hit the > I get the textual explain results but if I click the explain tab, nothing appears.

It is not clear if EXPLAIN is not working, or if this is a user error.
```

**Comment by eug3nix** [comment]
I have checked that and everything seems to be working just fine. The current interface may be a bit confusing and we should think in how improve that.
Here are some notes:
Just clicking on the explain tab does not run the explain feature for two reasons:
1. We have explain and explain/analyze options and it is not clear which one we should use when the tab is clicked
2. In general, Is it ok to eagerly run explain/explain analyze on the query when the explain tab is clicked?

Speaking of detecting explain explain analyze keywords in the submitted query and automatically opening the explain tab - this might be a good idea.

In general, to use the explain feature the user should:
- enter the query in the editor
- click the corresponding button on the toolbar

**Comment by eug3nix** [comment]
I guess the optimal solution would be to either move explain and explain analyze buttons to the explain tab itself OR have these buttons disabled unless Explain tab is selected

**Comment by eug3nix** [comment]
I have made several improvements to explain functionality:
explain and explain analyze buttons are now grouped together and this button group is separate from query buttons
these buttons are now disabled if explain tab is not active
when explain tab is selected, it now has a hint that the uset should click on explain or explain analyze button on the toolbar
if query entered by the user contains explain keyword, the explain tab will be opened automatically when user click the "Run query" button
if the query entered by the user contains explain keyword, clicking on explain/analyze button will no longer prepend the query with an extra explain keyword (previously this bug resulted in syntactically incorrect query)
This code is now merged into dev: [74f5114](https://github.com/commandprompt/pgmanage/commit/74f511419a6487d6db68409f8e4e7e0a7e577805)
